code/frameworks/angular/src/client/public-types.ts

```diff
@@ -9,7 +9,7 @@ import {
   StrictArgs,
   ProjectAnnotations,
 } from '@storybook/types';
-import { EventEmitter } from '@angular/core';
+import { EventEmitter, InputSignal } from '@angular/core';
```

Comment:
InputSignal might not be available in the `@angular/core` version range, which Storybook supports (Angular >= 15)

Reply:
You're right 😄 I'll look into this.

Best case we can define a type inside `Storybook` to act as a signal (this code will be a duplicate of the types Angular ships), worst case the peerDependency will have to be bumped to `>= 17.1`.

I'm assuming that bumping the peerDependency will be hard to achieve?

Reply:
Right! We will continue to support Angular 15 and upwards in Storybook 8.